geode-wan-txgrouping/src/distributedTest/java/org/apache/geode/internal/cache/wan/txgrouping/cli/commands/CreateTxGroupingGatewaySenderDUnitTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.internal.cache.wan.txgrouping.cli.commands;
+
+import static org.apache.geode.test.dunit.IgnoredException.addIgnoredException;
+
+import java.util.Properties;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.apache.geode.test.dunit.rules.ClusterStartupRule;
+import org.apache.geode.test.dunit.rules.MemberVM;
+import org.apache.geode.test.junit.assertions.CommandResultAssert;
+import org.apache.geode.test.junit.rules.GfshCommandRule;
+
+public class CreateTxGroupingGatewaySenderDUnitTest {
+
+  @Rule
+  public ClusterStartupRule cluster = new ClusterStartupRule();
+
+  @Rule
+  public GfshCommandRule gfsh = new GfshCommandRule();
+
+  MemberVM locator;
+
+  @Before
+  public void before() throws Exception {
+    locator = cluster.startLocatorVM(0);
+    cluster.startServerVM(1, new Properties(), locator.getPort());
+    gfsh.connectAndVerify(locator);
+  }
+
+  @Test
+  public void createTxGroupingParallelGatewaySender() {
+    addIgnoredException("could not get remote locator");
+
+    String createCommandString =
+        "create gateway-sender --id=sender1 --remote-distributed-system-id=1 --parallel --group-transaction-events=true";
```

Comment:
This doesn't seem modular yet. Is this testing deprecated `--group-transaction-events` and work is pending to add at `--type=TxGroupingParallelGatewaySender`?

Reply:
You can find an example of modularizing this in the first POC. Follow the new `type` property on the `create gateway-sender` command. 
https://github.com/pivotal-jbarrett/geode/blob/wip/wan-spi/geode-gfsh/src/main/java/org/apache/geode/management/internal/cli/commands/CreateGatewaySenderCommand.java#L83

Reply:
Done. We'll have to see how we reflect this in the documentation as we will need documentation specific for the module.